irr/src/CGLTFMeshFileLoader.cpp

```diff
@@ -0,0 +1,610 @@
+#include "CGLTFMeshFileLoader.h"
+
+#include "coreutil.h"
+#include "CSkinnedMesh.h"
+#include "ISkinnedMesh.h"
+#include "irrTypes.h"
+#include "IReadFile.h"
+#include "matrix4.h"
+#include "path.h"
+#include "S3DVertex.h"
+#include "quaternion.h"
+#include "vector3d.h"
+
+#include "tiniergltf.hpp"
```

Comment:
```suggestion
#include <tiniergltf.hpp>
```

Reply:
If we do this, I think we should treat it as one of the Irrlicht headers and not a vendored third-party lib.